test/visual/freedraw.js

```diff
@@ -27,178 +27,2225 @@
+      "y": 113.701
+    },
+    {
+      "x": 149.402,
+      "y": 111.709
+    },
+    {
+      "x": 150.398,
+      "y": 109.717
+    },
+    {
+      "x": 151.394,
+      "y": 108.721
+    },
+    {
+      "x": 153.386,
+      "y": 106.729
+    },
+    {
+      "x": 154.382,
+      "y": 103.741
+    },
+    {
+      "x": 157.371,
+      "y": 100.753
+    },
+    {
+      "x": 158.367,
+      "y": 99.757
+    },
+    {
+      "x": 159.363,
+      "y": 97.765
+    },
+    {
+      "x": 161.355,
+      "y": 94.777
+    },
+    {
+      "x": 163.347,
+      "y": 92.785
+    },
+    {
+      "x": 164.343,
+      "y": 91.789
+    },
+    {
+      "x": 165.339,
+      "y": 89.797
+    },
+    {
+      "x": 166.335,
+      "y": 86.809
+    },
+    {
+      "x": 167.331,
+      "y": 84.817
+    },
+    {
+      "x": 167.331,
+      "y": 82.825
+    },
+    {
+      "x": 167.331,
+      "y": 80.833
+    },
+    {
+      "x": 167.331,
+      "y": 78.841
+    },
+    {
+      "x": 167.331,
+      "y": 76.849
+    },
+    {
+      "x": 167.331,
+      "y": 74.857
+    },
+    {
+      "x": 166.335,
+      "y": 72.865
+    },
+    {
+      "x": 165.339,
+      "y": 71.869
+    },
+    {
+      "x": 164.343,
+      "y": 70.873
+    },
+    {
+      "x": 163.347,
+      "y": 68.881
+    },
+    {
+      "x": 162.351,
+      "y": 67.885
+    },
+    {
+      "x": 161.355,
+      "y": 66.889
+    },
+    {
+      "x": 160.359,
+      "y": 65.893
+    },
+    {
+      "x": 159.363,
+      "y": 64.897
+    },
+    {
+      "x": 158.367,
+      "y": 63.901
+    },
+    {
+      "x": 157.371,
+      "y": 62.905
+    },
+    {
+      "x": 156.375,
+      "y": 61.909
+    },
+    {
+      "x": 155.378,
+      "y": 60.913
+    },
+    {
+      "x": 154.382,
+      "y": 59.917
+    },
+    {
+      "x": 153.386,
+      "y": 58.921
+    },
+    {
+      "x": 152.39,
+      "y": 57.925
+    },
+    {
+      "x": 152.39,
+      "y": 55.933
+    },
+    {
+      "x": 152.39,
+      "y": 53.94
+    },
+    {
+      "x": 152.39,
+      "y": 51.948
+    },
+    {
+      "x": 153.386,
+      "y": 49.956
+    },
+    {
+      "x": 154.382,
+      "y": 48.96
+    },
+    {
+      "x": 154.382,
+      "y": 46.968
+    },
+    {
+      "x": 155.378,
+      "y": 45.972
+    },
+    {
+      "x": 156.375,
+      "y": 44.976
+    },
+    {
+      "x": 156.375,
+      "y": 42.984
+    },
+    {
+      "x": 157.371,
+      "y": 41.988
+    },
+    {
+      "x": 158.367,
+      "y": 39
+    },
+    {
+      "x": 159.363,
+      "y": 37.008
+    },
+    {
+      "x": 159.363,
+      "y": 35.016
+    },
+    {
+      "x": 160.359,
+      "y": 33.024
+    },
+    {
+      "x": 161.355,
+      "y": 31.032
+    },
+    {
+      "x": 161.355,
+      "y": 29.04
+    },
+    {
+      "x": 162.351,
+      "y": 27.048
+    },
+    {
+      "x": 162.351,
+      "y": 25.056
+    },
+    {
+      "x": 163.347,
+      "y": 24.06
+    },
+    {
+      "x": 164.343,
+      "y": 23.064
+    }
+  ];
```

Comment:
this pr seems all good, can you just keep this in a line for scrolling sanity only?
Even if we make a string and then we parse it each time.

Reply:
The IDE I use (VSCODE) allows me to collapse it with ease. That's one reason I changed it.
I prefer to extract it to a dedicated json file. Do you have any objections? It's better IMO. Once we migrate importing will be simple.

Reply:
i have VSCODE too, but yes it can be on its own file, too, as every fixture usually is.